### ISSUE TITLE
 nexa

### ISSUE THREAD

**Issue by sugak1** [bzminer/bzminer#183]
when amd card support on nexa

**Comment by MkrierPharmanity** [comment]
i dont thinks its possible, there is no amd nexa miner

**Comment by Jorgelote** [comment]
yes there is an amd nexa miner, it is wildrig miner, just that it was never optimized
